receiver/sqlqueryreceiver/config.go

```diff
@@ -80,6 +80,7 @@ type MetricCfg struct {
 	Aggregation      MetricAggregation `mapstructure:"aggregation"`
 	Unit             string            `mapstructure:"unit"`
 	Description      string            `mapstructure:"description"`
+	Tags             map[string]string `mapstructure:"tags"`
```

Comment:
Why do you think this functionality is essential part of the receiver? Addition of custom attributes can be easily achieved by using attributes processor https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/attributesprocessor

Reply:
Using the processor requires intercepting the metric by name and adding the tags. Totally doable, but seems way more intuitive to just be able to set static tag values at the receiver config. In our case we have 20+ queries all generating a fair amount of metrics and having the ability to set the config at the receiver is more maintainable then setting it at the processor. Not to mention the overhead is next to nothing to have this config option on the receiver.

Reply:
Makes sense. Thanks for providing the reasons. 

Let's just rename the config option to OTel terminology. Maybe `extra_attributes`, `metric_attributes` or just `attributes`?

Reply:
or `static_attributes`

Reply:
@mx-psi rational for config addition.